server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
I don't think meta alone would be correct here, according to wikipedia meta is "a prefix used in English to indicate a concept which is an abstraction from another concept, used to complete or add to the latter". [1] Meaning meta only really says nothing. Metadata would be correct here.

[1] https://en.wikipedia.org/wiki/Meta